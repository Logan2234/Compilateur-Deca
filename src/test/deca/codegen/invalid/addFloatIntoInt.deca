// Description:
//    Tentative d'addition d'un float et un int dans un int (impossible car nous ne pouvons pas transformer un float en int)
//
// Resultats:
//    ligne 13 : Erreur, pas possible de transformer un float en int
//
// Historique:
//    cree le 07/01/2023

{
	int i = 4;
	float j = 0.4;
	i = j + i;
	print(i);
}