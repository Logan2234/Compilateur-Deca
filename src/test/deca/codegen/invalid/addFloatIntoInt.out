  IMA -- Fichier ./src/test/deca/codegen/invalid/addFloatIntoInt.ass introuvable
